docs/introduction-to-smart-contracts.rst

```diff
@@ -84,27 +84,28 @@ registering with username and password — all you need is an Ethereum keypair.
 
     contract Coin {
         // The keyword "public" makes those variables
-        // readable from outside.
+        // easily readable from outside.
```

Comment:
Much like the above, I would say don't add 'easily'.

Reply:
I added it to stress that even without 'public', the variables are readable in principle

Reply:
@chriseth Ahhh, OK. I think there's a better way to put this, and as this is the 'getting started' guide simplicity is a strong focus. Stick with this for now and I'll think about something for a future PR.